Fixed some inaccuracies and typos. (#97)
Joystream/helpdesk: roles/council-members/README.md

- the `Candidacy Limit`

##### Scenario A:

> If: `Council Size <= Applicants <= Candidacy Limit`. I.e., the number of applicants is between the `Council Size` and `Candidacy Limit`.

Reviewer: ```suggestion
If: `Council Size <= Applicants <= Candidacy Limit`. I.e. the number of applicants is between the `Council Size` and `Candidacy Limit`.
```

The `Announcing` stage ends, with all the applicants proceeding to the `Voting` stage.

##### Scenario B:

> If: `Candidacy Limit < Applicants`. I.e., the number of applicants is greater than the `Candidacy Limit`.

Reviewer: ```suggestion
If: `Candidacy Limit < Applicants`. I.e. the number of applicants is greater than the `Candidacy Limit`.
```

All of the applicants are sorted and ranked by their stake. The `Announcing` stage ends, and only those with a rank better than, or equal to, the `Candidacy Limit` will proceed to the `Voting` stage.

The applicants that did not make it to the `Voting` stage get their stake back right away.

##### Scenario C:

> If: `Applicants < Council Size`. I.e., the number of applicants is smaller than the `Council Size`.

Reviewer: ```suggestion
If: `Applicants < Council Size`. I.e. the number of applicants is smaller than the `Council Size`.
```

  - you use the same key, computer and browser - without clearing local storage.
  - if you save the `Random salt`, you only need the key
- if you voted for one or more active CMs, you can "re-use" your voting stake from the last election cycle

> - this means that you if you staked 10k JOY voting a one or more current CMs, and have 5k "free" balance, you can stake 15k for voting

Reviewer: ```suggestion
- this means that you if you staked 10k JOY voting for one or more current CMs, and have 5k "free" balance, you can stake 15k for voting
```

  - What you have done to prepare
  - Any questions about the role
  - etc.

> - Longtime community members

Reviewer: ```suggestion
- Long-term community members
```

An example of the structure for a single Council KPI is outlined below. Note that the number of KPIs, success events, individual and sum of the rewards, and complexity of the KPIs per term will vary.

Council KPI - Term `X`

> - `id:` - The unique identifier (e.g., `X.1`)

Reviewer: ```suggestion
- `id:` - The unique identifier (e.g. `X.1`)
```

As managing the Bounties will be a significant part of their (potential) KPI rewards, they have some excess resources and incentives to hire the "right" people. They can either pay for this out of pocket, request funding for this through spending proposals, reserve some part of the Bounty reward for this purpose or by asking Jsgenesis for designated funding.

> In the end, the Council will still be responsible for the work the BM does, so they should still verify the spending and/or text proposals made, and seek feedback of the BMs performance (e.g., responsiveness and communication) from participants.

Reviewer: ```suggestion
In the end, the Council will still be responsible for the work the BM does, so they should still verify the spending and/or text proposals made, and seek feedback of the BM's performance (e.g. responsiveness and communication) from participants.
```

If the BM needs clarity or more information than what is covered in the Bounty Issue, reply to it for clarity.

> Once the draft is completed, the BM creates a text or spending proposal (if funding is needed for their tasks) presenting a draft of the Forum post that will be made for the bounty. If the text is too long to fit in the proposal, a link the text is shared in its place.

Reviewer: ```suggestion
Once the draft is completed, the BM creates a text or spending proposal (if funding is needed for their tasks) presenting a draft of the Forum post that will be made for the bounty. If the text is too long to fit in the proposal, a link to the text is shared in its place.
```

5. The Bounty Manager, when performing their weekly grading and reporting, can look through the Content Curators Forum thread, and see what has been approved or not. If a video/channel posted in 2. is not approved, it need not be graded. Once the BM has completed grading all (approved) entries, they create a report containing:
  - For each Entrant identified by their membershipId, membershipHandle and account
    - all channels and videos by their IDs

>     - with grading (i.e., reward) for each video ID

Reviewer: ```suggestion
    - with grading (i.e. reward) for each video ID
```

  - A summary of the rewards
  - A list of all videos/channels that was not approved by Curators

> 6. This report is submitted as a PR to the Community Repo, along with a Spending Proposal equaling the sum of the Rewards (plus, if applicable, a payment for their own labor).

Reviewer: ```suggestion
6. This report is submitted as a PR to the Community Repo, along with a Spending Proposal equalling the sum of the Rewards (plus, if applicable, a payment for their own labour).
```
Leaving this, since it's just EN-GB vs EN-US

  - entrants the BM picks as winners

1. The Bounty Manager sets some Bounty specific rules (in addition to the [General Steps](#general-steps)), such as:

>   - Where and how a Banner should be submitted initially (e.g., the forum or Community Repo)

Reviewer: ```suggestion
  - Where and how a Banner should be submitted initially (e.g. the forum or Community Repo)
```

  - Exact specifications of a banner (e.g. file format(s), resolutions, etc.)
  - Information on how to find resources (e.g., "blank canvas", fonts, preferred themes, etc.)
  - Deadline(s) for submissions
  - Reserves the right to extend deadlines under certain conditions
  - etc.
2. Entrants sees the Bounty, and creates banners.
3. After doing so, they post in the bounties Forum post (and/or makes a draft PR to the Community Repo), with their work.

> 4. The BM performs their reviews, and selects up to three Candidates (if there are fewer than three, the BM can choose to extend the deadline). These are encouraged to make spending proposals.

Reviewer: ```suggestion
4. The BM performs their reviews, and selects up to three Candidates (if there are fewer than three, the BM can choose to extend the deadline). These are each encouraged to make spending proposals.
```

    - Experience with using cryptocurrencies
    - Milestones and timelines
    - Brief explanation of why they should be Assigned the Bounty

>   - The BM is free to suggest or set parameters for milestones (e.g., 3), timelines (e.g., max two weeks for each), and other reporting requirements

Reviewer: ```suggestion
  - The BM is free to suggest or set parameters for milestones (e.g. 3), timelines (e.g. max two weeks for each), and other reporting requirements
```

    - A milestone implies some partial delivery at an agreed time, with some part of the compensations being paid out on completion.
2. Applicants apply for the Bounty. (They may also ask questions, and set some conditions if applicable.)
3. After the application stage is over, the Bounty Manager either:
  - Assigns an Applicant themselves, or makes a text proposal to the Council proposing to assign one (depending on what was agreed in [initially](#general-steps))

>   - Extend the Application process - possible with some new terms (e.g., requesting more funding, longer deadlines, etc.)

Reviewer: ```suggestion
  - Extend the Application process - possible with some new terms (e.g. requesting more funding, longer deadlines, etc.)
```

#### Workflow
The workflow will depend both on the Reward Distribution and the [Format](#format), and must be established beforehand.

> - For "Closed" formats, an Applier must present a bid because they should be assigned the given Bounty. This should include detailed terms, such as time needed, costs, etc. If approved, this makes the terms valid.

Reviewer: ```suggestion
- For "Closed" formats, an Applier must present a bid justifying why they ought to be assigned the given Bounty. This should include detailed terms, such as time needed, costs, etc. If approved, this makes the terms valid.
```
